strings/src/main/res/values-zu/strings.xml

```diff
@@ -383,4 +389,14 @@
   <!--##############################################
     # Permissions
     ##############################################-->
+  <!--An entity is any person, place or thing that a form can be about. This title appears on an experimental settings screen to view entities managed in the current project.-->
```

Comment:
Does that make sense to have comments like here in translations (languages other than English)? If someone knows English they can check them in the main strings file. Isn't it difficult then to put new translations into right place (below a proper comment)?

Reply:
You should check out Transifex! Those comments provide context in the UI. Without context, many strings are impossible to translate. In general, we should always add this kind of context to new translation strings.

Reply:
Oh, and to be explicit, all of these files other than `strings.xml` are auto-generated by Transifex. By default it passes the comments through. That makes them usable in another translation system (e.g. Android Studio's) in case we ever change the Transifex workflow.

Reply:
<img width="506" alt="Screen Shot 2022-11-23 at 11 11 59 AM" src="https://user-images.githubusercontent.com/967540/203628642-bb9c5288-63e9-429a-a3a5-a88ec48fe89d.png">

From https://www.transifex.com/getodk/collect/translate/#pl_PL/strings/442049170?q=translated%3An+text%3Aentities

Reply:
Ah, got it, sorry I misunderstood. I thought you knew that but figured maybe you forgot! :D Yeah, I think it makes sense to pull whatever Transifex gives us which is this.